test/RiskManagerV1.test.js

```diff
@@ -27,6 +28,12 @@ describe("RiskManagerV1", () => {
     testToken = await TestToken.deploy()
     await testToken.deployed()
 
+    const SignerBondsProcessorStub = await ethers.getContractFactory(
+      "SignerBondsProcessorStub"
```

Comment:
Same comment here about the stub, I'd be fine with using just `SignerBondsEscrow` 🤷

Reply:
See https://github.com/keep-network/coverage-pools/commit/56e45d3fdb69f813aacf8f66f630e6288358f3ec